backend/apps/owasp/management/commands/owasp_sync_posts.py

```diff
@@ -0,0 +1,41 @@
+"""A command to update OWASP posts from owasp.org data."""
+
+import requests
+import yaml
+from django.core.management.base import BaseCommand
+
+from apps.github.utils import get_repository_file_content
+from apps.owasp.models.post import Post
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        posts_api_url = "https://api.github.com/repos/OWASP/owasp.github.io/contents/_posts"
```

Comment:
Please use pygithub for GH API interactions when possible. 

Also use `from apps.github.utils import get_repository_file_content`